manufacturers/Watteco/models/Vaqao/uplink_decoder.js

```diff
@@ -0,0 +1,41 @@
+let watteco = require("../../decode.js")
+let ngsild = require("../../ngsi-ld.js")
+
+let batch_param = [3, [{taglbl: 0,resol: 1, sampletype: 4,lblname: "occupancy", divide: 1, unit: ""},
+    { taglbl: 1, resol: 10, sampletype: 7,lblname: "temperature_1", divide: 100, unit: "CEL"},
+    { taglbl: 2, resol: 100, sampletype: 6,lblname: "humidity_1", divide: 100, unit: "P1"},
+    { taglbl: 3, resol: 10, sampletype: 6,lblname: "CO2", divide: 1, unit: "52"},
+    { taglbl: 4, resol: 10, sampletype: 6,lblname: "TVOC", divide: 1, unit: ""}]];
+
+let endpointCorresponder = {
+    concentration: ["TVOC", "CO2"],
+    temperature: ["temperature_1","temperature_2"],
+    humidity: ["humidity_1","humidity_2"],
+    pin_state:["violation_detection"]
+}
+
+function main() {
+    var payload = process.argv[3];
+    var time = process.argv[4];
+    // ********* Test pattern (uncomment to test behaviour) ********************
+        // Pattern batch
+        // payload = "424500000111008FC85EC128B4872088AD060442001E3322118A98782B";
+        // Pattern "uplink standard report containing firmware version"
+        // payload = "110100000002000D0305020015E2"
+        // Pattern "uplink standard report containing humidity value"
+        // payload = "110A04050000210E89"
+        // Pattern "uplink standard report containing dataup info" (IGNORED) 
+        payload = "110180040000000800"
```

Comment:
This should be commented, right ? And I tried to execute the script with one of the test pattern but nothing was returned.

Reply:
Yes, indeed

Reply:
corrected in last update